### ISSUE TITLE
ArgumentError in spi/tooling_jobs#update

### ISSUE THREAD

**Issue by iHiD** [exercism/website#1156]
## Error in v3

**ArgumentError** in **spi/tooling_jobs#update**
malformed format string - %C

[View on Bugsnag](https://app.bugsnag.com/exercism/v3/errors/60d5b880cf450f00071ed26f?event_id=60d5b880007f7a25ae920000&i=gh&m=ci)

## Stacktrace

    app/models/submission/analysis.rb:52 - %
    app/models/submission/analysis.rb:52 - block in comments
    app/models/submission/analysis.rb:43 - map
    app/models/submission/analysis.rb:43 - comments
    app/models/submission.rb:119 - analyzer_feedback
    app/models/iteration.rb:11 - analyzer_feedback
    app/serializers/serialize_iteration.rb:24 - call
    app/channels/solution_channel.rb:13 - block in broadcast!
    app/channels/solution_channel.rb:13 - map
    app/channels/solution_channel.rb:13 - broadcast!
    app/models/solution.rb:181 - broadcast!
    app/models/iteration.rb:55 - broadcast!
    app/commands/submission/representation/process.rb:32 - call
    app/commands/tooling_job/process.rb:18 - process_representer_job!
    app/commands/tooling_job/process.rb:8 - call
    app/controllers/spi/tooling_jobs_controller.rb:4 - update

[View full stacktrace](https://app.bugsnag.com/exercism/v3/errors/60d5b880cf450f00071ed26f?event_id=60d5b880007f7a25ae920000&i=gh&m=ci)

*Created automatically via Bugsnag*

**Comment by iHiD** [comment]
@exercism/elixir This is an issue with the analyzer:
```json
"{\"comments\":[{\"comment\":\"elixir.pacman-rules.use_strictly_boolean_operators\",\"type\":\"essential\"}],\"summary\":\"Check the comments for things to fix. 🛠\"}"},
```

I'm guessing there's some parse error in the file? Maybe something not escaped?

**Comment by iHiD** [comment]
Yeah - the `%` needs to be escape in the URL of # of https://raw.githubusercontent.com/exercism/website-copy/main/analyzer-comments/elixir/pacman-rules/use_strictly_boolean_operators.md

**Comment by iHiD** [comment]
An error linked to this issue has been reopened in Bugsnag
[**Octokit::UnprocessableEntity** in **spi/tooling_jobs#update**](https://app.bugsnag.com/exercism/v3/errors/60d5b880cf450f00071ed26f?event_id=618e76f400897641928f0000&i=gh&m=ra)

**Comment by neenjaw** [comment]
Do you know which exercise it was analyzing

**Comment by angelikatyborska** [comment]
My bet would be on this rust comment from @ErikSchierboom  that got merged at the same time as this issue got reopened: https://github.com/exercism/website-copy/pull/2128/files#diff-e0c4882cc1dd17448fa1fd4de20cb7649d3347918b0f272b8272791e9bae955fR3

**Comment by ErikSchierboom** [comment]
@iHiD Should it be escaped as `%%`?

**Comment by iHiD** [comment]
Yes. I believe so.

**Comment by ErikSchierboom** [comment]
[PR submitted](https://github.com/exercism/website-copy/pull/2133)
